### PR TITLE
Update install docs for Android to link to APK

### PR DESCRIPTION
**Pull request by hamishwillee**
- [ ] Once this is merged will need to fix up qgroundcontrol.com links

### CODE REVIEW

File: en/getting_started/download_and_install.md [hunk 1]
```diff
@@ -56,15 +56,20 @@ To install *QGroundControl* for Ubuntu Linux 16.04 LTS or later:
...
 
-## Android
+## Android {#android}
 
 Install *QGroundControl* for Android 5.1 or later:
 
-1. Open the Google Play Store [QGroundControl link](https://play.google.com/store/apps/details?id=org.mavlink.qgroundcontrol).
-1. Follow the installation instructions.
+1. Download and install the stable build: [QGroundControl.apk](https://qgroundcontrol.s3-us-west-2.amazonaws.com/latest/QGroundControl.apk)
+   - Android phones have different ways of locating downloaded files.
```
Comment (hamishwillee) [inline en/getting_started/download_and_install.md]: FYI only, too many possible ways to explain. This at least encourages people to understand they need to look for a file manager or whatever download system is used on their device.

**Comment by DonLakeFlyer** [comment]
In both stable and daily build places I would say something like: "QGroundControl is temporarily unavailable from the Google Play Store. We are working on a fix this, but it may take some time. Until then you can download the APK directly from here for installation:" or something along those lines. I wouldn't go into detail about the why 64 bit thing.

Then I would change the download link on qgroundcontrol.com front page to point to the download and install page.

**Comment by hamishwillee** [comment]
Cool. All done.

**Comment by DonLakeFlyer** [comment]
Thanks @hamishwillee

**Pull request merged**